javaslang-benchmark/src/test/java/javaslang/collection/ArrayBenchmark.java

```diff
@@ -77,43 +127,51 @@ public void tearDown() {
         }
 
         @Benchmark
-        public void java_mutable(Initialized state) {
+        public Object java_mutable(Initialized state) {
```

Comment:
This is a trick that JMH suggests to avoid Dead Code Elimination, as @paplorinc mentioned.
The other option would be to use `Blackholes`

[http://hg.openjdk.java.net/code-tools/jmh/file/13ae22114e22/jmh-samples/src/main/java/org/openjdk/jmh/samples/JMHSample_08_DeadCode.java](http://hg.openjdk.java.net/code-tools/jmh/file/13ae22114e22/jmh-samples/src/main/java/org/openjdk/jmh/samples/JMHSample_08_DeadCode.java)